crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -1844,6 +1844,31 @@ _.tt.tttt.t.tt.ttttt_
             -_-
 ENDMAP
 
+#### A dangerous temple to kiku
+NAME: beargit_kiku_chapel
+DEPTH: Swamp:2-
+TAGS: no_monster_gen
+KMONS: 1 = bunyip zombie w:10 / bunyip skeleton w:10 / bog body w:20 / death scarab w:1
+KMONS: 2 = patrolling juggernaut zombie w:20 / patrolling caustic shrike zombie w:3 / patrolling shard shrike zombie w:3 / patrolling spectral juggernaut w:5 / patrolling spectral caustic shrike w:1
+KMONS: 4 = patrolling lich w:1 / patrolling fenstrider_witch; quick blade ego:pain w:1
```

Comment:
A lich is also on the spooky side for swamp, consider a necromancer instead.
```suggestion
MONS: patrolling necromancer / patrolling fenstrider_witch; quick blade ego:pain
```

Reply:
you're not going to let me get any kills!